### DISCUSSION TITLE
myNoise as music provider

### DISCUSSION THREAD

**Discussion by Kahros** [music-assistant/support#3340] [Feature requests and ideas]
I'm not sure how easy/hard this would be considering the website and app have many features involving playback, such as real time sound modulation to mute or heighten certain tones or frequencies, by maybe allowing for basic audio sound playback?

website: https://mynoise.net/

**Comment by YourAutisticLife** [comment]
I agree. I currently play Ambi Nature Radio through RadioBrowser, but it would be nice if I could use mynoise.net when I take a nap. Even if it played the default streams served by the site, that would be something.

**Reply by OzGav** [reply]
Looking at his FAQs it doesn't seem like he is keen to provide access to others...

**Comment by simbalasim** [comment]
Bump!

myNoise would be a perfect next level addition for the new **[audio overlay feature](https://github.com/music-assistant/server/pull/4674)**,  in the genre of the new providers [Rainy Mood ](https://github.com/music-assistant/server/pull/3844) and [Local Ambient Noise Loops ](https://github.com/music-assistant/server/pull/4676)

myNoise sounds are professionally handcrafted, quality wise in my opinion miles ahead than most other "noise"-webites. 
And the library is huge, has wide variety,  is constant growing, 

Even only the possibility to kind of embed the myNois.net-Webiste in MusicAssistant/HomeAssistant so that webistes audio could be used would be win. Right now myNoise.net blocks iframe embedding.

**Reply by digitalTARDIS2** [reply]
I wonder if Stéphane would be interested in collaborating with someone to build a MyNoise provider for Music Assistant, given that Stéphane’s audio stems and sliders are proprietary.  His FAQ mentions keeping the project exclusive for personal, non-commercial use. I wonder if a provider built with or for him—maybe restricted to active MyNoise patrons—could align with his terms while still letting him maintain full control over how his sounds are delivered.  Just throwing the idea out there to see if anyone has reached out to him before or if there’s interest in exploring a collaboration.
